jsonschema/src/keywords/format.rs

```diff
@@ -401,7 +414,8 @@ pub(crate) fn compile<'a>(
             "idn-email" => Some(IDNEmailValidator::compile(context)),
             "idn-hostname" if draft_version == Draft::Draft7 => {
                 Some(IDNHostnameValidator::compile(context))
-            }
+            },
+            "uuid" => Some(UUIDValidator::compile(context)),
```

Comment:
Please, put it behind `#[cfg(feature = "draft201909")]` macro & a condition as some other Draft 2019-09 validators in this match

Reply:
It might look like this:

```rust
#[cfg(feature = "draft201909")]
"uuid" if draft_version == Draft::Draft201909 => {
    Some(UUIDValidator::compile(context))
}
```

Reply:
yes but then my test fails because in test context there is not draft_version configuration

Reply:
You need to run tests with `--all-features` or `--features draft201909`

Reply:
ive updated my tests to use the tests_util file. im not sure the implementation is working correctly

```
        let passing_instance = json!("f308a72c-fa84-11eb-9a03-0242ac130003");
        let failing_instance = json!("º∆ª©");

        tests_util::is_valid(&schema, &passing_instance);
        tests_util::is_not_valid(&schema, &failing_instance);
```

"is not valid" makes the test fail, because i expect uuid::fromStr to return an error (and it does) but i think compile isnt calling my implementation correctly ?

Reply:
basically no matter what string i feed it, is_valid is always true

Reply:
yea when i debug, its not calling the UUIDValidator method i implemented :(

Reply:
Oh, I forgot to mention, but you need to use `JSONSchema::options().with_draft(Draft::Draft201909)` and then validate input, as this format is only available under this draft

Reply:
aha interesting.. so the test will need to change once/if the draft is removed?

i hit the debug point tho !